homework-css-selectors/nav-bar/style.css

```diff
@@ -0,0 +1,8 @@
+
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: yellow;
+}
+  
```

Comment:
Foarte buna varianta cu `display: flex`. Good job!

Ideea era sa folosesti in principiu hint-ul din descrierea temei (verticle-align: middle) ca sa vezi ca se poate si fara flexbox 😄